google/cloud/language/v1beta1/language_gapic.yaml

```diff
@@ -3,7 +3,7 @@ language_settings:
   java:
     package_name: com.google.cloud.language.spi.v1beta1
   python:
-    package_name: google.cloud.language.v1beta1
```

Comment:
Also, we call https://github.com/googleapis/gax-python [`google-gax`](https://pypi.python.org/pypi/google-gax) in Python-land.